README.md

```diff
@@ -90,199 +111,97 @@ NOTE: es6 imports won't work in browser or with bundlers which expect es6 module
-- **interaction** — Adds support for interactive features of fabric — selecting/transforming objects/groups via mouse/touch devices.
-- **parser** — Adds support for `fabric.parseSVGDocument`, `fabric.loadSVGFromURL`, and `fabric.loadSVGFromString`
-- **image_filters** — Adds support for image filters, such as grayscale of white removal.
-- **easing** — Adds support for animation easing functions
-- **node** — Adds support for running fabric under node.js, with help of [jsdom](https://github.com/tmpvar/jsdom) and [node-canvas](https://github.com/learnboost/node-canvas) libraries.
-- **freedrawing** — Adds support for free drawing
-- **erasing** — Adds support for object erasing using an eraser brush
-- **gestures** — Adds support for multitouch gestures with help of [Event.js](https://github.com/mudcube/Event.js)
-- **object_straightening** — Adds support for rotating an object to one of 0, 90, 180, 270, etc. depending on which is angle is closer.
-- **animation** — Adds support for animation (`fabric.util.animate`, `fabric.util.requestAnimFrame`, `fabric.Object#animate`, `fabric.Canvas#fxCenterObjectH/#fxCenterObjectV/#fxRemove`)
-
-Additional flags for build script are:
-
-- **requirejs** — Makes fabric requirejs AMD-compatible in `dist/fabric.js`. _Note:_ an unminified, requirejs-compatible version is always created in `dist/fabric.require.js`
-- **no-strict** — Strips "use strict" directives from source
-- **no-svg-export** — Removes svg exporting functionality
-- **sourcemap** - Generates a sourceMap file and adds the `sourceMappingURL` (only if uglifyjs is used) to `dist/fabric.min.js`
-
-For example:
-
-    node build.js modules=ALL exclude=json no-strict no-svg-export
 
-## Goals
+---
 
-- Unit tested (1150+ tests at the moment, 79%+ coverage)
-- Modular (~60 small ["classes", modules, mixins](http://fabricjs.com/docs/))
-- Cross-browser
-- [Fast](https://github.com/kangax/fabric.js/wiki/Focus-on-speed)
-- Encapsulated in one object
-- No browser sniffing for critical functionality
-- Runs under ES5 strict mode
-- Runs on a server under [Node.js](http://nodejs.org/) (active stable releases and latest of current) (see [Node.js limitations](https://github.com/kangax/fabric.js/wiki/Fabric-limitations-in-node.js))
-- Follows [Semantic Versioning](http://semver.org/)
+## Other Solutions
 
-## Supported browsers
+| Project                                                  | Description          | Demo |
+| -------------------------------------------------------- | -------------------- | :--: |
+| [Three.js](https://github.com/mrdoob/three.js/)          | 3D graphics          |
+| [PixiJS](https://github.com/pixijs/pixijs)               | WebGL renderer       |
+| [Konva](https://github.com/konvajs/konva)                | _Competition_        |  ❌  |
+| [Canvas2PDF](https://github.com/joshua-gould/canvas2pdf) | PDF renderer         |
+| [html-to-image](https://github.com/bubkoo/html-to-image) | HTML to image/canvas |
```

Comment:
i m pretty much sure if we want to list other solution we should expand the description a tiny bit.
Especially for Konva we should say `Library with similar features and a competitor in the space`
What is the red X under the demo column?

Reply:
I want to add demos for each 
Not for konva
I thought adding the three.js demo @melchiar did and adapt the pdf one i did

Reply:
I haven't added the three.js demo yet due to our lack of SSL (codepen requires the model file to be hosted at a secure source). Unless we can find a good sandbox solution for running demos

Reply:
Codesandbox?

Reply:
You can view #8135 there are ready to edit demos

Reply:
Does codesandbox resolve the asset storage issue?

Reply:
Maybe...